theme/DesignTheme/Views/NewsItem.scss

```diff
@@ -0,0 +1,22 @@
+.full-width {
+  position: relative;
+  right: 50%;
+  left: 50%;
+  width: 100vw !important;
+  max-width: initial !important;
+  margin-right: -50vw !important;
+  margin-left: -50vw !important;
+  object-fit: cover;
+  height: 480px;
+}
```

Comment:
Questo lo metterei in un qualche file scss più generico, non è a disposizione solo delle news come classe